README.md

```diff
@@ -32,7 +30,7 @@ This is a simple math magic application built with JavaScript, HTML, and CSS. It
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍